.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] Nice to add the `node_modules` folder in the `.gitignore` 👌, but regrading to the requirements you should also add `test.md`.

![1](https://user-images.githubusercontent.com/45231276/174903490-ccca7e70-b64b-4348-a3a6-2f8263bdcace.PNG)